### DISCUSSION TITLE
Is there an option to disable -v4 and -v13 suffix?

### DISCUSSION THREAD

**Discussion by Thyristor-Chopper** [iBotPeaches/Apktool#3646] [Q&A]
It's so hassle to rename these to remove these suffixes every time I decompile APKs but I can't find such an option in the help text.

![sddsds](https://github.com/user-attachments/assets/a0ba8fdd-66f1-4ae5-8ce1-b5e50d2a12a1)

**Comment by iBotPeaches** [comment]
There is not - those are implicit qualifiers required.
